tests/jballerina-unit-test/src/test/resources/test-src/object/object-referencing-module-variable.bal

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2023 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Need to use the LLC header, without "All Rights Reserved".

Reply:
Removed this file